Proposals/BTC Forks Roadmap Proposal.md

```diff
@@ -85,91 +48,78 @@ A decentralised system works best when all its components are in equilibrium.
+**Our Options**
 
+While there are many different proposals on how to solve this problem, most solutions boil down to one of two solutions. One solution is to change the economic model of bitcoin so that the bitcoin network does not scale much further on-chain, but rather exra layers are developed on top of the bitcoin network. These extra layers would connect to bitcoin for security but would not have the same security model of bitcoin. The Lightning is an example of a layer 2 network. This solution could be boiled down to "Bitcoin: An Electronic Settlement Network". Another solution is to allow the bitcoin network to scale on-chain by allowing more transactions to happen directly on the bitcoin network. This solution would keep the economic model of bitcoin that has worked over the past 7 years and would extend it to the future. "This solution can be boiled down to "Bitcoin: A Peer-to-Peer Electronic Cash Network for Payments and Settlement Transactions"
 
-**5.iii. Consensus Abstraction Layers (CAL)**
+Currently, the development of the bitcoin reference client is done by the group of bitcoin developers called "Core". While there are various other smaller development teams that offer their own bitcoin clients, the Core developers control the development of the client that most network participants are aware of. The Core developers of bitcoin have decided to use the solution of "Bitcoin: An Electronic Settlement Network". A significant (although not necessarily a majority) number of the network participants do not feel that this solution is the best way forward.
 
-The consensus abstraction layers are a way of assessing and tracking changes to a cryptocurrency made by developers. These changes may be the general principles that the participants follow, the consensus logic that the network follows or specific changes in the code of an implementation. Use of the consensus abstraction layers will allow participants to hold other participants to account within the ecosystem and will reinforce the ability for users to consent to the network(s) they are using. If an implementation/code change is made that is not inline with the current abstraction layers of that cryptocurrency then this implementation is considered to be forked and the type of fork is dictated by the CAL that the fork has taken place on.
+**Reasoning for Not Choosing the Opposing Solution**
 
-<p align="center">
-  <img src="http://i.imgur.com/AVQRBKP.png"/>
-</p>
+Our reasoning for the decision that the "Bitcoin: An Electronic Settlement Network" is not an optimal solution is that, from our analysis it seems that the economic and security model of this solution does not work and that the solution does not use fully understood technology. 
```

Comment:
If we claim it does not work the analysis has to be very rigorous. It's like opponents of Bitcoin saying "Bitcoin does not work" - they need to provide rigorous proof.

What is definitely true is that the "Electronic Settlement" model imposes an economic change on the system, and the implications of that - not least of all on the fee reward model of Bitcoin mining - are poorly understood, as are the risks related to possible re-introduction of intermediaries (large hubs). I guess that could be seen as part of the security of Bitcoin.

And it's not only the case that LN uses "not fully understood technology" - it is a solution postulated on some currently *unsolved* problems (routing) - i.e. for these problems no convincing design of a solution even exists.